types/ts4.1/type-specs.tsx

```diff
@@ -59,7 +59,7 @@ const invalidParamsWithGeneric: Params<{ id: number }> = { id: 13 }; // $ExpectE
   This is a <b>mixed</b> content
 </Route>;
 
-<Route path="/users/:id">{(params: Params): React.ReactNode => `User id: ${params.id}`}</Route>;
+<Route path="/users/:id">{(params: Params) => `User id: ${params.id}`}</Route>;
```

Comment:
This one is interesting. Because we use React 17 for testing the types, the two `ReactNode` versions were conflicting with each other. 

This could lead to some cases where a user decides to define their function explicitly returning `ReactNode` and they might run into this issue. But I suppose since it was a bug they, unfortunately, would have to fix it themselves.